docs/introduction-to-smart-contracts.rst

```diff
@@ -208,7 +208,8 @@ The functions that make up the contract, and that users and contracts can call a
 The ``mint`` function sends an amount of newly created coins to another address.
 The :ref:`require <assert-and-require>` function call defines conditions that reverts all changes if not met.
 In this example, ``require(msg.sender == minter);`` ensures that only the creator of the contract can call ``mint``,
-and ``require(amount < 1e60);`` ensures a maximum amount of tokens. This ensures that there are no overflow errors in the future.
+and ``require(amount < 1e60);`` ensures balance isn't overflowed by any feasible number of calls (``uint``
+maximum value exceeds 1e77.)
```

Comment:
What about something like "ensures a maximum amount of tokens: The function can be called very often, but due to gas costs, there is still a practical limit."?

Reply:
I like that as a turn away from overflow concerns and toward how the economy works - there's no room for a primer on how to handle error/overflow in smart contracts anyway. I would still keep the tip about how this value compare to maximum uint value though. Let me revise.